test/e2e/serial/tests/sched_removal.go

```diff
@@ -81,11 +85,11 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources scheduler remova
 		It("[case:2][test_id:49093][tier1] should keep new scheduled workloads pending", func() {
 			var err error
 
-			By(fmt.Sprintf("deleting the NRO Scheduler object: %s", nroSchedObj.Name))
-			err = fxt.Client.Delete(context.TODO(), nroSchedObj)
+			By(fmt.Sprintf("deleting the NRO Scheduler object: %s", serialconfig.Config.NROSchedObj.Name))
+			err = fxt.Client.Delete(context.TODO(), serialconfig.Config.NROSchedObj)
 			Expect(err).ToNot(HaveOccurred())
 
-			dp := createDeployment(fxt, "testdp", schedulerName)
+			dp := createDeployment(fxt, "testdp", serialconfig.Config.SchedulerName)
 
```

Comment:
why not use serialconfig schedobj in line 112 (and all its instances)? I see it is used similarly there as well

Reply:
this is a good point. At this point in time I'm doing 1:1 replacement, in the future we would want to simplify/reorganize in the future